### PR TITLE
Get started update

### PR DESCRIPTION
**Pull request by nermiller**
Adds snap and appimage instructions to Linux
Updates About

### CODE REVIEW

File: docs/stack/about/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+description: Learn more about Redis Stack
+weight: 1
+---
+
+Redis Stack is an improved starting point for working with Redis. We've bundled together the best of the technology that we have to offer into an easy 
+to use package. Redis Stack extends the core capabilities of Redis OSS and provides a complete developer experience for debugging and more.
+
+In addition to all of the features of Redis OSS, Redis Stack supports:
+
+* Probabilistic data structures
```
Comment (chayim) [inline docs/stack/about/index.md:13]: Should these link to JSON + friends, instead of "in stack" below?

File: docs/stack/about/index.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+## Why Redis Stack?
+
+Redis Stack was created to allow developers to build real-time applications with a backend data platform that can reliably process requests in 
+milliseconds or less. Redis Stack does this by taking the original Redis OSS as the core and enhancing it with modern data models, data processing tools,
+and continuing to fight complexity at every turn. Ultimately, the goal of Redis Stack is to build a real-time data platform that continues to fulfill the
+philosophy of Redis OSS: simplicity, performance, and reliability. 
+
+Redis Stack unifies and simplifies the developer experience of the leading Redis modules and the capabilities they provide. Redis Stack bundles 
+five Redis modules: [RedisJSON](/docs/stack/json), [RediSearch](/docs/stack/search), [RedisGraph](/docs/stack/graph),
```
Comment (chayim) [inline docs/stack/about/index.md:27]: ```Redis Stack bundles the following capabilities into redis: ...```

This way you don't have to maintain "five" as we add more (gears).

File: docs/stack/about/index.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+## Redis Stack packaging
+
+There are three distinct Redis Stack packages to choose from:
+
+* **Redis Stack Server**:  This packages contains Redis OSS and module extensions only. It does *not* contain Redis Insight, the developer desktop
+application. This package is best for production deployment and is intended to be a drop-in replacement (for example, if you're already deploying 
+Redis OSS as a cache).
+
+* **Redis Stack (Desktop)**: This package contains everything a developer needs in a single bundle. This includes Redis OSS and module extensions
```
Comment (chayim) [inline docs/stack/about/index.md:42]: I think it's RedisInsight - [here](https://redis.com/redis-enterprise/redis-insight/), but I could be out of date.

File: docs/stack/about/index.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+{{% alert title="Note" color="warning" %}}
+ 
+Modules as described here are generic, and we list only the representative use cases.
+
+{{% /alert %}}
+
+## Clients
+
+Several Redis client libraries support Redis Stack. These include [redis-py](https://redis-py.readthedocs.io/en/stable/), [node_redis](https://github.com/redis/node-redis), and [Jedis](https://github.com/redis/jedis). In addition, four higher-level object
```
Comment (chayim) [inline docs/stack/about/index.md:61]: nredisstack

File: docs/stack/about/index.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+* Graph data models with the Cypher query language
+
+## Why Redis Stack?
+
+Redis Stack was created to allow developers to build real-time applications with a backend data platform that can reliably process requests in 
+milliseconds or less. Redis Stack does this by taking the original Redis OSS as the core and enhancing it with modern data models, data processing tools,
+and continuing to fight complexity at every turn. Ultimately, the goal of Redis Stack is to build a real-time data platform that continues to fulfill the
+philosophy of Redis OSS: simplicity, performance, and reliability. 
+
+Redis Stack unifies and simplifies the developer experience of the leading Redis modules and the capabilities they provide. Redis Stack bundles
```
[the remark this answers is not in the document]
Comment (nermiller) [reply docs/stack/about/index.md:26]: @K-Jo do we call them "capabilities" or "features" from now on?

**Review by K-Jo — CHANGES_REQUESTED** [review]

File: docs/stack/about/index.md (hunk 3 above)
Comment (nermiller) [reply docs/stack/about/index.md:42]: @K-Jo I had to port your changes into this PR, but it should be accurate now.

File: docs/stack/about/index.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+Redis Stack is made up of several components, licensed as follows:
+
+* Redis Stack Server, which combines open source Redis with RediSearch, RedisJSON, RedisGraph, RedisTimeSeries, and RedisBloom, is dual-licensed under the Redis Source Available License ([RSALv2]((/docs/stack/license/)) and the [Server Side Public License](https://en.wikipedia.org/wiki/Server_Side_Public_License) (SSPL). A breakdown of licensing by Stack component versions is shown in the table below. For more information about Redis licensing, see [Licenses](https://redis.com/legal/licenses/).
+
+* RedisInsight is licensed under the SSPL.
+
+### Versions and licenses
+
+Starting November 15, 2022, our default binary distributions of Redis Stack and our Redis modules have been dual-licensed under RSALv2 and SSPL. When using the source code, you can apply either RSALv2 or SSPLv1.
```
[the remark this answers is not in the document]
Comment (nermiller) [reply docs/stack/about/index.md:88]: @K-Jo the detail was covered in the section above. Made a smallish update to this wording.

File: docs/stack/about/index.md (hunk 1 above)
Comment (nermiller) [reply docs/stack/about/index.md:12]: @chayim individual module sections are already linked elsewhere

File: docs/stack/about/index.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+* **Redis Stack Server**:  This package contains Redis OSS and module extensions only. It does *not* contain RedisInsight, the developer desktop
+application. This package is best for production deployment and is intended to be a drop-in replacement (for example, if you're already deploying 
+Redis OSS as a cache).
+
+* **Redis Stack (Desktop)**: This package contains everything a developer needs in a single bundle. This includes Redis OSS and module extensions 
+along with the RedisInsight desktop application. If you want to create an application locally and explore how it interacts with Redis, this is 
+the package for you.
+
+* **RedisInsights**: This package contains _only_ RedisInsight, the developer desktop application. If you've used Redis Stack Server and
```
Comment (nermiller) [inline docs/stack/about/index.md:44]:
```suggestion
* **RedisInsight**: This package contains _only_ RedisInsight, the developer desktop application. If you've used Redis Stack Server and 
```

File: docs/stack/about/index.md [hunk 8]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
 Redis beyond caching, Redis Stack is the place to start. Redis Stack delivers the core capabilities developers love about Redis and goes beyond 
 to help you build modern applications where performance is paramount. 
 
+## RedisInsight
+
+The best way to get started with Redis Stack is with [RedisInsight](https://docs.redis.com/latest/ri/), a visualization tool for understanding and optimizing Redis data.
+
+RedisInsight allows you to:
+* View data structures visually with the Browser tool and gropu keys according to their namespaces
```
Comment (nermiller) [inline docs/stack/about/index.md:37]:
```suggestion
* View data structures visually with the Browser tool and group keys according to their namespaces.
```

File: docs/stack/about/index.md [hunk 9]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
...
+* CRUD support for most Redis Stack data structures.
```
Comment (nermiller) [inline docs/stack/about/index.md:38]:
```suggestion
* Use CRUD most Redis Stack data structures.
```

File: docs/stack/about/index.md [hunk 10]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
...
+* Workbench: an advanced command line interface with intelligent command auto-complete and complex data visualizations.
```
Comment (nermiller) [inline docs/stack/about/index.md:39]:
```suggestion
* Leverage Workbench, an advanced command-line interface with intelligent command auto-complete and complex data visualizations.
```

File: docs/stack/about/index.md [hunk 11]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
...
+* Analyze traffic to Redis in real time with the Profiler tool
```
Comment (nermiller) [inline docs/stack/about/index.md:40]:
```suggestion
* Analyze traffic to Redis in real time using the Profiler tool.
```

File: docs/stack/about/index.md [hunk 12]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
...
+* A CLI tool is accessible at any time within the application
```
Comment (nermiller) [inline docs/stack/about/index.md:41]:
```suggestion
* Use embedded Redis CLI at any time. 
```

File: docs/stack/about/index.md [hunk 13]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
...
+
+The best way to get started with Redis Stack is with [RedisInsight](https://docs.redis.com/latest/ri/), a visualization tool for understanding and optimizing Redis data.
+
+RedisInsight allows you to:
+* View data structures visually with the Browser tool and gropu keys according to their namespaces
+* CRUD support for most Redis Stack data structures.
+* Workbench: an advanced command line interface with intelligent command auto-complete and complex data visualizations.
+* Analyze traffic to Redis in real time with the Profiler tool
+* A CLI tool is accessible at any time within the application 
+* Analyze memory usage with the Memory Analysis tool
```
Comment (nermiller) [inline docs/stack/about/index.md:42]:
```suggestion
* Analyze memory usage with the Memory Analysis tool.
```

File: docs/stack/about/index.md [hunk 14]
```diff
@@ -29,21 +29,31 @@ The overarching vision is to provide developers with a powerful platform for all
...
+* Identify and troubleshoot bottlenecks with the Slowlog tool
```
Comment (nermiller) [inline docs/stack/about/index.md:43]:
```suggestion
* Identify and troubleshoot bottlenecks using the Slowlog tool.
```

**Review by nermiller — COMMENTED** [review]
A few minor edits, @K-Jo

**Review by K-Jo — DISMISSED** [review]

File: docs/stack/about/index.md [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+Redis beyond caching, Redis Stack is the place to start. Redis Stack delivers the core capabilities developers love about Redis and goes beyond 
+to help you build modern applications where performance is paramount. 
+
+## RedisInsight
+
+The best way to get started with Redis Stack is with [RedisInsight](https://docs.redis.com/latest/ri/), a visualization tool for understanding and optimizing Redis data.
+
+RedisInsight allows you to:
+* View data structures visually with the Browser tool and group keys according to their namespaces.
+* Use CRUD most Redis Stack data structures.
```
Comment (nermiller) [inline docs/stack/about/index.md:38]:
```suggestion
* Use CRUD on most Redis Stack data structures.
```

**Comment by nermiller** [comment]
@K-Jo please give me one more check. I fixed my own typo :)

**Review by K-Jo — APPROVED** [review]

**Pull request merged**